charts/ocis/templates/collaboration/deployment.yaml

```diff
@@ -0,0 +1,119 @@
+  template:
+    {{- include "ocis.templateMetadata" (dict "scope" $ "configCheck" false) | nindent 4 }}
+    spec:
+      {{- include "ocis.affinity" $.Values.services.collaboration | nindent 6 }}
+      {{- include "ocis.securityContextAndtopologySpreadConstraints" $ | nindent 6 }}
+      {{- include "ocis.priorityClassName" $.priorityClassName | nindent 6 }}
+      {{- include "ocis.hostAliases" $ | nindent 6 }}
+      nodeSelector: {{ toYaml $.nodeSelector | nindent 8 }}
+      containers:
+        - name: {{ $.appName }}
+          {{- include "ocis.image" $ | nindent 10 }}
+          command: ["ocis"]
+          args: ["collaboration", "server"]
+          {{- include "ocis.containerSecurityContext" $ | nindent 10 }}
+          env:
+            {{- include "ocis.serviceRegistry" $ | nindent 12 }}
+
+            - name: COLLABORATION_LOG_COLOR
+              value: {{ $.Values.logging.color | quote }}
+            - name: COLLABORATION_LOG_LEVEL
+              value: {{ $.Values.logging.level | quote }}
+            - name: COLLABORATION_LOG_PRETTY
+              value: {{ $.Values.logging.pretty | quote }}
+
+            - name: COLLABORATION_TRACING_ENABLED
+              value: "{{ $.Values.tracing.enabled }}"
+            - name: COLLABORATION_TRACING_TYPE
+              value: {{ $.Values.tracing.type | quote }}
+            - name: COLLABORATION_TRACING_ENDPOINT
+              value: {{ $.Values.tracing.endpoint | quote }}
+            - name: COLLABORATION_TRACING_COLLECTOR
+              value: {{ $.Values.tracing.collector | quote }}
+
+            - name: COLLABORATION_DEBUG_PPROF
+              value: "{{ $.Values.debug.profiling }}"
+
+            - name: COLLABORATION_GRPC_ADDR
+              value: 0.0.0.0:9301
+            - name: COLLABORATION_HTTP_ADDR
+              value: 0.0.0.0:9300
+            - name: COLLABORATION_DEBUG_ADDR
+              value: 0.0.0.0:9304
+
+            - name: COLLABORATION_APP_NAME
+              value: {{ $officeSuite.name | quote }}
+
+            - name: COLLABORATION_APP_LOCKNAME
+              value: com.github.owncloud.collaboration-{{ $officeSuite.name }}
+
+            - name: COLLABORATION_APP_INSECURE
+              value: {{ $officeSuite.insecure | quote }}
+
+            - name: COLLABORATION_CS3API_DATAGATEWAY_INSECURE
+              value: {{ $.Values.insecure.ocisHttpApiInsecure | quote }}
+
+            - name: COLLABORATION_APP_ICON
+              value: {{ $officeSuite.iconURI | quote }}
+
+            - name: COLLABORATION_APP_ADDR
+              value: {{ $officeSuite.uri | quote }}
+
+            - name: COLLABORATION_WOPI_SRC
+              {{- if $officeSuite.ingress.enabled }}
+              value: https://{{  $officeSuite.ingress.domain }}
+              {{- else }}
+              value: http://{{ $.appName }}.{{ template "ocis.namespace" $ }}.svc.cluster.local:9300
+              {{- end }}
+
+            - name: COLLABORATION_JWT_SECRET
```

Comment:
TODO: should this be the same as OCIS_JWT_SECRET ? Currently it's a different secret